src/components/menus/content-contextual-menu.js

```diff
@@ -86,6 +93,103 @@ const ContentContextualMenu = (props) => {
         setOpenDialog(dialogId);
     };
 
+    const duplicateItem = () => {
+        if (activeElement) {
+            const duplicateSuffix = '(1)';
+            elementExists(
```

Comment:
Maybe later, add an api in directory server to generate a new unique name, and use it here ...